packages/splits-sdk/src/client/splitV2.ts

```diff
@@ -621,6 +623,52 @@ export class SplitV2Client extends SplitV2Transactions {
   async owner(split: Address): Promise<Address> {
     return this._getSplitV2Contract(split).read.owner()
   }
+
+  async getSplitMetadata(splitAddress: Address): Promise<{ split: SplitV2 }> {
+    this._requirePublicClient()
+
+    const [updateLogs, createLogs, owner] = await Promise.all([
+      this._publicClient?.getLogs({
```

Comment:
we'll want to set the start block to the factory deployment block (for both `getLogs` calls)